astrocut/footprint_cutouts.py

```diff
@@ -0,0 +1,325 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""This module creates cutouts from data cubes found in the cloud."""
+
+import json
+import os
+import re
+from typing import List, Union
+import warnings
+from threading import Lock
+
+from astropy.coordinates import SkyCoord
+from astropy.table import Table, Column
+import astropy.units as u
+from astropy.utils.exceptions import AstropyWarning
+from cachetools import TTLCache, cached
+import fsspec
+import numpy as np
+import requests
+from spherical_geometry.polygon import SphericalPolygon
+
+from astrocut.exceptions import InvalidQueryError
+from astrocut.cube_cut import CutoutFactory
+
+from .utils.utils import parse_size_input
+
+TESS_ARCSEC_PER_PX = 21  # Number of arcseconds per pixel in a TESS image
+FFI_TTLCACHE = TTLCache(maxsize=10, ttl=900)  # Cache for FFI footprint files
+CUBE_CUT_THREADS = 8  # Number of threads to use in `cube_cut` function
+
+
+def _s_region_to_polygon(s_region: Column):
+    """
+    Takes in a s_region string of type POLYGON or CIRCLE and returns it as
```

Comment:
i don't think this docstring is correct -- this function as currently written only supports `POLYGON`, not `CIRCLE`

Reply:
It looks like this docstring is still referring to CIRCLE but that doesn't look supported

Reply:
Are you looking at the most recent commit? I do see the change on that docstring.